src/clj/game/cards/events.clj

```diff
@@ -609,7 +609,8 @@
              :effect (effect (access-bonus :rd (max 0 (min 4 (available-mu state)))))}]})
 
 (defcard "Déjà Vu"
-  {:prompt "Choose a card to add to Grip" :choices (req (cancellable (:discard runner) :sorted))
+  {:req (req (not (zone-locked? state :runner :discard)))
+   :prompt "Choose a card to add to Grip" :choices (req (cancellable (:discard runner) :sorted))
```

Comment:
mind splitting this onto two lines for me? it's a hold over from very old code, and i'm not a fan.